src/components/selection-field.jsx

```diff
@@ -41,8 +40,14 @@ export default function SelectionField({
 }) {
   const selfRef = useRef(null);
   const [showFilterbox, setShowFilterbox] = useState(false);
-  const model = useModel(app, createDefinition(field));
-  const layout = useLayout(model);
+  const [model, modelError] = useModel(app, createDefinition(field));
+  if (modelError) {
```

Comment:
Stylewise, I would've gone with a one-liner on these to keep the logic a bit more condensed so it's easier to read `if (modelError) throw modelError;`

Just an idea: What about having a error-throwing hook for all of these?

Like:

```js
const [model] = useErrorThrow(useModel(app, createDefinition(field));`
```